README.md

```diff
@@ -55,38 +55,33 @@ Binding to a C implementation of the cblas interface (e.g. ATLAS, OpenBLAS, Inte
 
 The recommended (free) option for good performance on both Linux and Darwin is OpenBLAS.
 
-### blas/dbw
+### blas/blas64
 
 Wrapper for an implementation of the double precision real (i.e., `float64`) part
 of the blas API
 
-You have to register an implementation before you can use the BLAS functions:
-
 ```Go
 package main
 
 import (
 	"fmt"
 
-	"github.com/gonum/blas/cgo"
-	"github.com/gonum/blas/dbw"
+	"github.com/gonum/blas/blas64"
 )
 
-func init() {
-	dbw.Register(cgo.Implementation{})
-}
-
 func main() {
-	v := dbw.NewVector([]float64{1, 1, 1})
-	fmt.Println("v has length:", dbw.Nrm2(v))
+	v := blas64.Vector{Inc: 1, Data: []float64{1, 1, 1}}
+	fmt.Println("v has length:", blas64.Nrm2(len(v.Data), v))
 }
 ```
 
-### blas/zbw
+### blas/cblas128
 
 Wrapper for an implementation of the double precision complex (i.e., `complex128`)
 part of the blas API
 
+Currently blas/cblas128 requires blas/cgo and must be built with the `cblas` build tag.
```

Comment:
Could you add a "use the -tags flag" or something like that? It's surprisingly hard to find information on how to build with a tag.